python3: can't open file '/root/projects/Bubbles-payload/funpumper/metrics_enricher_loop': [Errno 2] No such file or directory
  File "/root/projects/Bubbles-payload/funpumper/metrics_enricher_loop.py", line 41
    pass pass
IndentationError: expected an indented block after 'with' statement on line 39
  File "/root/projects/Bubbles-payload/funpumper/metrics_enricher_loop.py", line 86
    pass
    ^
IndentationError: expected an indented block after 'try' statement on line 85
  File "/root/projects/Bubbles-payload/funpumper/metrics_enricher_loop.py", line 91
    pass
    ^
IndentationError: expected an indented block after 'try' statement on line 90
